src/pages/my-profile/screens/MyProfile/MyProfile.scss

```diff
@@ -4,10 +4,12 @@
     padding-top: 2.4rem;
     overflow-y: auto;
     max-height: calc(100vh - 19rem);
+    min-height: calc(100vh - 19rem);
 
     @include mobile {
         padding: 0;
         max-height: calc(100vh - 11rem);
+        min-height: calc(100vh - 11rem);
```

Comment:
can we not give height instead?

Reply:
nope, i tried that but it messes with the height of my profile
![Screenshot 2024-07-05 at 9 55 40 AM](https://github.com/deriv-com/p2p/assets/103412909/33077b83-3acd-456b-9e60-5c44e447f02e)